Dockerfile

```diff
@@ -1,14 +1,19 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.6-alpine3.7
+FROM tiangolo/uwsgi-nginx-flask:python3.8-alpine
 
-ENV STATIC_URL /static
-ENV STATIC_PATH /var/www/app/static
+COPY . /app
 
-COPY ./requirements.txt /var/www/requirements.txt
+COPY docker/.aws /root/.aws
+COPY docker/sqlitebackup.sh /usr/local/bin/
+COPY docker/supervisor.d/ /etc/supervisor.d/
+COPY docker/prestart.sh /app/.
 
-COPY . /app
-RUN pip install -r /var/www/requirements.txt
+RUN chmod 755 /usr/local/bin/sq*
+
+RUN apk add bash sqlite
+RUN pip3 install awscli awscli_plugin_endpoint
 
-EXPOSE 8080
+RUN cd /usr/local/bin && wget https://raw.githubusercontent.com/fluential/docker-sqlite-to-s3/master/sqlite-to-s3.sh && chmod 755 sqlite*
```

Comment:
On README.md:106 I see you reference `https://github.com/jacobtomlinson/docker-sqlite-to-s3` but then use a fork of it here? Since this fork is identical to the original repo, could we use the file from the repo referenced in the README?

Reply:
Good catch, will fix